SCons/Tool/MSCommon/common.py

```diff
@@ -389,12 +401,16 @@ def get_output(vcbat, args=None, env=None, skip_sendtelemetry=False):
         'VSCMD_DEBUG',   # enable logging and other debug aids
         'VSCMD_SKIP_SENDTELEMETRY',
         'windir', # windows directory (SystemRoot not available in 95/98/ME)
+        'VCPKG_DISABLE_METRICS',
+        'VCPKG_ROOT',
```

Comment:
Can we move this list to be a package variable? 

That way it can be changed from a SConscript/SConstruct if desired..

Reply:
Absolutely

Reply:
How would that work? Seems a bit awkward to have to deal with a variable in the vastly more common default case - we've only had the one issue/PR relating to someone trying to use vcpkg, so it can't be very common in the world SCons reaches.

Reply:
@mwichmann I believe that @bdbaddog is talking about moving the constant list of variables from inside the function to file-level.  This way it could be modified externally if desired but not necessarily recommended.

Reply:
Pushed change.

Reply:
ah... "package variable" is a defined term in SCons (https://scons.org/doc/production/HTML/scons-man.html#v-PackageVariable) so I thought that's what it meant.  If the intent was module global, yes, that's a good idea: things that are "constants" (even if they're changeable) are better at the top of the file than embedded in a function.

Reply:
It was only moved immediately above the function using the list rather than the top of the file.  It is only used in one place (Captain Obvious).

Hope that is ok.

Reply:
Personally prefer them at the top for visibility, but there's some argument that something with only one use is also good kept close to the place of use. There are few absolutes with Python.

Reply:
Agreed. better to have all the constants at the top of the file if possible.

Reply:
None of the current constants in `MSCommon/common.py` are at the top of the file.  All inline near use point.